### PR TITLE
dashboard: Improve Wireguard and IPsec Widgets

### PR DESCRIPTION
**Pull request by Monviech**
Fixes: https://github.com/opnsense/core/issues/7739

### CODE REVIEW

File: src/opnsense/www/js/widgets/IpsecLeases.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ export default class IpsecLeases extends BaseTableWidget {
         // Prepare a summary row for user counts
         let userCountsRow = `
             <div>
-                <span><b>${this.translations.users}:</b> ${totalUsersCount} - <b>${this.translations.online}:</b> ${onlineUsersCount} - <b>${this.translations.offline}:</b> ${offlineUsersCount}</span>
+                <span>${this.translations.users}: ${totalUsersCount} | ${this.translations.online}: ${onlineUsersCount} | ${this.translations.offline}: ${offlineUsersCount}</span>
```
Comment (fichtner) [inline src/opnsense/www/js/widgets/IpsecLeases.js:114]: With all of HTML and CSS at our disposal we still use "-" and "|" to delimit row data? :)

File: src/opnsense/www/js/widgets/Metadata/Core.xml [hunk 2]
```diff
@@ -252,12 +252,12 @@
             <offline>Offline</offline>
             <total>Tunnels</total>
             <unconfigured>IPsec is currently disabled. Click to configure IPsec.</unconfigured>
-            <notunnels>There are currently no tunnels.</notunnels>
+            <notunnels>Phase1 unconfigured</notunnels>
```
Comment (fichtner) [inline src/opnsense/www/js/widgets/Metadata/Core.xml:255]:
```suggestion
            <notunnels>Phase1 not configured</notunnels>
```

**Review by fichtner — APPROVED** [review]
ok, though structuring rows with characters seems a bit last century :D

File: src/opnsense/www/js/widgets/IpsecLeases.js (hunk 1 above)
Comment (Monviech) [reply src/opnsense/www/js/widgets/IpsecLeases.js:114]: I mean we could use some font awesome icon there as well but which one would look nice?

Comment (Monviech) [reply src/opnsense/www/js/widgets/IpsecLeases.js:114]: Here are some suggestions:

https://fontawesome.com/icons/ellipsis-vertical?s=solid
https://fontawesome.com/icons/grip-lines-vertical?f=classic&s=solid
https://fontawesome.com/icons/minus?f=classic&s=solid
https://fontawesome.com/icons/grip-vertical?f=classic&s=solid

Any of them in your favor or do you have a different suggestion?

Comment (fichtner) [reply src/opnsense/www/js/widgets/IpsecLeases.js:114]: column-based data should be divs or tables but I don't want to derail this PR

Comment (Monviech) [reply src/opnsense/www/js/widgets/IpsecLeases.js:114]: Lets fix this later then. It has to be thought of in multiple widgets.

**Review by swhite2 — APPROVED** [review]
Looks good!

I went ahead and adjusted the widgets to do a left-header style table, also removed the `endCommandTransition` in the ipsec tunnels widget to give more instant button feedback, since a tunnel can be started immediately after. This is not the case for `OpenVPN clients` as this is simply a kill command.

**Comment by SeimusS** [comment]
Already tested on production unit. Works and looks beautiful thank you both +1

Regards,
S.

**Pull request merged**